### ISSUE TITLE
Code for FSAR mandatory figure? 

### ISSUE THREAD

**Issue by clumb** [pbs-assess/csasdowntex#248]
Hi,

Thank you so much for all of the work on csasdown! It is amazing to have rmarkdown templates for reproducible CSAS reports.

I'm not sure if this is the right place to post this question, as it's not an issue or a bug. 

I work for DFO in Winnipeg, in the Arctic and Aquatic Research Division. We're having a stock assessment for Great Slave Lake (Lake Whitefish, Lake Trout and Inconnu), so we're going to draft fsars for the meeting.

I'm wondering if R code might be available for the mandatory panel of plots for FSAR Figure 1 (image below)? Obviously, no worries if it's not, I just thought I'd ask. I noticed from an issue posted in November that work on the fsar template is underway.

I'm fairly new to using csasdown. A couple weeks ago, I cloned csasdown (version 0.1.3) and the fsar branch to save locally on my machine. I found an image of the mandatory figure (in folders fsar -> knitr-figs-docx -> base1-1.png). I couldn't find the code that generated the figure. My apologies if the code is there and I missed it.

Thank you for your help, and for csasdown!

Chelsey 

![image](https://github.com/pbs-assess/csasdown/assets/90855580/01989835-2141-444a-b3c7-84b1e66a29a1)

**Comment by seananderson** [comment]
The fsar template is still in development. @ricardd has the most applied experience trying to use it. I'm also not sure where that code went. See the [other thread](https://github.com/pbs-assess/csasdown/issues/242). We're thinking about a good central place to put that code still. It's in the Git history for sure, but that's not very convenient.

**Comment by clumb** [comment]
Hi Sean,

Thanks very much for your quick reply and for reminding me about the specs from Julie for the plots in the other issues thread. That's a big help!

Chelsey

**Comment by ricardd** [comment]
@clumb on the fsar branch, see the function definition that creates a simulated dataset here:

[R/fake-data.R](../blob/fsar/R/fake-data.R)

and the function that creates a 4-panel figure using the simulated dataset:

[R/plot-timeseries.R](../blob/fsar/R/plot-timeseries.R)

these two functions are called when one renders the example FSAR 

https://github.com/pbs-assess/csasdown/blob/fsar/inst/rmarkdown/templates/fsar/skeleton/01_fsar.Rmd#L99-L104

**Comment by clumb** [comment]
Hi Daniel,

Thank you so much for your reply and for the links! I copied the .R files and saved them in my local RProject folder. I was able to get the figure generated and render the fsar (yay!).

I'll have to figure out why I didn't have copies of the .R files. From the links you provided, it seems there are folders and files that aren't in my local repo, even though I cloned the repo from github. For now at least I have a working fsar template.

Thank you again for all of your help!

Chelsey
